lib/inertia_rails/renderer.rb

```diff
@@ -6,16 +6,6 @@
 
 module InertiaRails
   class Renderer
-    attr_reader(
-      :component,
-      :configuration,
-      :controller,
-      :props,
-      :view_data,
-      :encrypt_history,
-      :clear_history
-    )
```

Comment:
These are public but they aren't referenced anywhere else in the codebase and I can't imagine that someone is calling them in the wild somewhere. Please someone sanity check me on that!

Reply:
I hope you're right, but this is a breaking change right? Could they be used in tests?

Reply:
The rspec helpers mock it: https://github.com/inertiajs/inertia-rails/blob/99774c86480f4b4f696d2b41b486565b20be5194/lib/inertia_rails/rspec.rb#L85 but it looks like you're right that they don't care about these particular vars.

Reply:
that's mocking the _arguments_ to `.new` rather than the accessor methods